tests/test_json_model.py

```diff
@@ -143,8 +144,21 @@ def test_user_agent(self):
         headers, path_params, query_params, body)
 
     self.assertEqual(headers['user-agent'],
-        'my-test-app/1.23.4 google-api-python-client/' + __version__ +
-        ' (gzip)')
+        'my-test-app/1.23.4')
+
+  def test_x_goog_api_client(self):
+    model = JsonModel(data_wrapper=False)
+
+    headers = {'x-goog-api-client': 'gccl/my-test-client/1.23.4'}
```

Comment:
Use a standard form (e.g. `'gccl/1.23.4'`) instead of this nonstandard form, to avoid confusion.